pytorch_lightning/accelerators/ddp_base_backend.py

```diff
@@ -58,7 +58,8 @@ def test_step(self, args):
         return output
 
     def barrier(self, name: str = None):
```

Comment:
again unused name argument

Reply:
this class is getting dropped in a few PRs.